src/panels/config/automation/trigger/types/ha-automation-trigger-device.ts

```diff
@@ -159,12 +164,27 @@ export class HaDeviceTrigger extends LitElement {
     });
   }
 
-  private _extraFieldsComputeLabelCallback(localize) {
+  private _extraFieldsComputeLabelCallback(localize, trigger: DeviceTrigger) {
     // Returns a callback for ha-form to calculate labels per schema object
-    return (schema) =>
+    return (schema): string =>
       localize(
         `ui.panel.config.automation.editor.triggers.type.device.extra_fields.${schema.name}`
-      ) || schema.name;
+      ) ||
+      localize(
+        `component.${trigger.domain}.device_automation.extra_fields.${schema.name}`
```

Comment:
I think it is already loaded since the backends `strings.json`s
```json
{
  "device_automation": {
    "condition_type": {...}
    "trigger_type": {...}
  }
}
keys are already loaded too. I'm not sure where this happens though 😬.